docs/application/native/guides/multimedia/media-controller.md

```diff
@@ -196,11 +336,242 @@ To process the received command on the server side:
    ret = mc_server_set_playback_state_command_received_cb(g_mc_server, command_received_cb, NULL);
    ```
 
-4. When no longer needed, destroy the media controller server handle using the `mc_server_destroy()` function:
+4. Destroy the media controller server handle using the `mc_server_destroy()`, when media controller server handle is no longer needed:
+
+   ```
+   mc_server_destroy(g_server_h);
+   ```
+
+<a name="send_cmd"></a>
```

Comment:
As mentioned earlier, please avoid using HTML code. Use only MD specific code.

Reply:
The HTML tag changed to head anchor for MD.